airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/check/commands.py

```diff
@@ -77,10 +76,10 @@ async def python(pipeline_context: ClickPipelineContext) -> CommandResult:
 @check.command(cls=DaggerPipelineCommand)
 @pass_pipeline_context
 async def java(pipeline_context: ClickPipelineContext) -> CommandResult:
-    """Format java, groovy, and sql code via spotless."""
+    """Format java code via spotless in maven."""
```

Comment:
The SQL formatting has been removed while you were OOO. "Removed" is a strong word considering that the formatter we had reaaaaaally didn't do much except format whitespace. I spent the better part of a work day looking for better alternatives and didn't find anything suitable to our needs. It turns out that SQL has a zillion dialects and if you want effective formatting you need to be able to parse it meaningfully, which is hard to do. Considering we actually have few SQL files I came to the conclusion that it just wasn't worth the trouble.

The groovy formatting indeed disappears with this PR but here again that formatter just didn't do that much. I've written far more groovy than I'd like these past few months and I literally can't remember a single instance where the formatter did anything meaningful. Actually, now I'll double check if that's the result of a bad config or something. If so I'll add the groovy config to the pom.xml file and to build.gradle. Everything gradle does with spotless, maven can do too.

Reply:
I checked and I can confirm that the groovy formatter is mostly useless but I'll add it anyway.

Reply:
Actually I'll add it but as a no-op and I'll change the config to actually make it do things in a separate PR, because that will trigger CI for multiple connectors.

Reply:
That works for me. Thanks for the context and for doing so!